shell/platform/darwin/ios/framework/Headers/FlutterEngine.h

```diff
@@ -172,16 +175,20 @@ FLUTTER_EXPORT
  * it will signal the engine to stop animations and drawing.  However, neither will impact the state
  * of the Dart program's execution.
  */
-@property(nonatomic, weak) FlutterViewController* viewController;
+@property(nonatomic, weak, nullable) FlutterViewController* viewController;
```

Comment:
I believe `weak` is always nullable even inside `NS_ASSUME_NONNULL_BEGIN`, you don't need to decorate this one.

Reply:
ah good to know, thanks